crawlers/ecolex/.gitignore

```diff
@@ -0,0 +1,12 @@
+jurisprudence/*
```

Comment:
This `.gitignore` values should be added to the main `.gitignore`. Afterwards, remove the file.